### PR TITLE
pace device-steps for slow qemu

### PR DESCRIPTION
**Pull request by eriknordmark**
@deitch This makes things more sane (less likely to watchdog) when I run under qemu

### CODE REVIEW

File: pkg/pillar/scripts/device-steps.sh [hunk 1]
```diff
@@ -45,6 +45,25 @@ while [ $# != 0 ]; do
...
+wait_for_touch() {
+    if [ "$1" = "lisp-ztr" ]; then
+	return
+    fi
+    f=/var/run/"$1".touch
+    waited=0
+    while [ ! -f "$f" ] && [ "$waited" -lt 60 ]; do
+	    echo "$(date -Ins -u) waiting for $f"
+	    sleep 3
+	    waited=$((waited + 3))
```
Comment (gkodali-zededa) [inline pkg/pillar/scripts/device-steps.sh]: Yetus would complain here. I think the $ can be removed.

**Comment by eriknordmark** [comment]
> So it just slows things down, so slow startup time won’t lead to watchdog blowing up?
> 
> It just feels like it shouldn’t be necessary, but let’s try and see.

It paces the startup of the different agents so they don't all content with eachother during the startup. Seems to just take a few seconds for each under qemu, so doesn't get near any watchdog timeout.

File: pkg/pillar/scripts/device-steps.sh (hunk 1 above)
Comment (eriknordmark) [reply pkg/pillar/scripts/device-steps.sh]: Actually the $ is needed (in /bin/sh and in bash). No yetus complaints (apart from the tabs which I just fixed)

Comment (gkodali-zededa) [reply pkg/pillar/scripts/device-steps.sh]: Yes, you are right. I got confused.

**Pull request merged**